errors_test.go

```diff
@@ -1,38 +1,51 @@
 package opslevel_test
 
 import (
+	"fmt"
 	"testing"
 
 	ol "github.com/opslevel/opslevel-go/v2025"
 	"github.com/rocktavious/autopilot/v2023"
 )
 
-func TestFormatErrorsWorks(t *testing.T) {
+func TestHasAPIErrorsWorks(t *testing.T) {
 	// Arrange
 	errs := []ol.Error{
 		{Message: "can't be blank", Path: []string{"resource", "id"}},
 		{Message: "is not a valid input", Path: []string{"id"}},
 	}
 	// Act
-	output := ol.FormatErrors(errs)
+	output := ol.HasAPIErrors(errs)
 	// Assert
+	autopilot.Equals(t, true, ol.ErrIs(output, ol.ErrorAPIError))
 	autopilot.Equals(t, `OpsLevel API Errors:
 	- 'resource.id' can't be blank
 	- 'id' is not a valid input`,
 		output.Error())
 }
 
-func TestFormatErrorsNoPath(t *testing.T) {
+func TestHasAPIErrorsNoPath(t *testing.T) {
 	// Arrange
 	errs := []ol.Error{
 		{Message: "can't be blank", Path: []string{"base"}},
 		{Message: "is not a valid input", Path: []string{""}},
 	}
 	// Act
-	output := ol.FormatErrors(errs)
+	output := ol.HasAPIErrors(errs)
 	// Assert
+	autopilot.Equals(t, true, ol.ErrIs(output, ol.ErrorAPIError))
 	autopilot.Equals(t, `OpsLevel API Errors:
 	- '' can't be blank
 	- '' is not a valid input`,
 		output.Error())
 }
+
+func TestIsResourceFoundError(t *testing.T) {
```

Comment:
This is the main goal - we wanted an easy way in downstream tools to beable to detect this kind of error cleanly and consistently.  It will reduce 100's lines of error checking across the tools.